### ISSUE TITLE
Ability to configure cpu-cfs-quota and cpu-cfs-quota-period for kubelet

### ISSUE THREAD

**Issue by hrzbrg** [kubernetes/kops#5826]
**1. Describe IN DETAIL the feature/behavior/change you would like to see.**

There have been discussions in [1] and [2] to modify the default period of 100ms to avoid issues with excessive throttling. Some users came to the conclusion that setting `cpu-cfs-quota=false` actually is better than any quota [3]. So it would be great have both of these parameters under control in kops as well.
 
[1] https://github.com/kubernetes/kubernetes/issues/51135
[2] https://github.com/kubernetes/kubernetes/pull/63437
[3] https://twitter.com/mikkeloscar/status/1037443700513796103

**Comment by hrzbrg** [comment]
PR is up: https://github.com/kubernetes/kops/pull/6375

**Comment by hrzbrg** [comment]
PR: #6375

**Comment by joekohlsdorf** [comment]
You should already be able to set these parameters without further changes in kops:
```
spec:
  kubelet:
    cpu-cfs-quota: "true"
    cpu-cfs-quota-period: "5ms"
```

I haven't tried it because these flags are available beginning with Kubernetes 1.12 and kops only supports 1.11.

**Comment by hrzbrg** [comment]
closed as #6375 is merged
